site/src/js/index.js

```diff
@@ -52,7 +52,8 @@ function handleTags() {
 
     Array.from(contentEl.querySelectorAll('.item')).forEach(el => {
         const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false);
-        if (!show) {
+
+        if (!show || (![...tags].includes("tag-expired") && el.classList.contains("tag-expired"))) {
```

Comment:
is the array copy really needed to call `includes`?

Reply:
This logic should be in the reduce logic because that's what's determining if the card should be visible or not 🤔

Reply:
The `expired`-logic shouldn't be in the `reduce` since it only needs to happen once. I'll move it to line 54 though, since the `expired` filtering should affect `show`.

Reply:
i.e.

```js

const show = tags.reduce(fn, tags.includes('tag-expired') && el.classList.contains('tag-expired'))

```

Reply:
You're right 🤔 we'll table this for now - if we come up with something better by the time this is ready to be merged, we can come back to it, otherwise, we can merge with this 🙂